exe/ruby-lsp

```diff
@@ -21,7 +22,7 @@ end
 require_relative "../lib/ruby_lsp/internal"
 
 if ARGV.include?("--debug")
-  sockets_dir = "/tmp/ruby-lsp-debug-sockets"
+  sockets_dir = "#{Dir.tmpdir}/ruby-lsp-debug-sockets"
```

Comment:
We can't change it in this way. Otherwise, the clients ([console](https://github.com/Shopify/ruby-lsp/blob/main/bin/rdbg#L15), [VS Code](https://github.com/Shopify/vscode-ruby-lsp/blob/main/src/debugger.ts#L210)) won't be able to pick the socket from the right folder.

Reply:
Good point, but shouldn't those be changed to read `$TMPDIR`?

Reply:
Windows sets 
TEMP=C:\Users\jan\AppData\Local\Temp
TMP=C:\Users\jan\AppData\Local\Temp

So clients on Windows can easily pick up this folder.

For non Windows clients this will be most probably /tmp.
Not changing it, is like not running on Windows. There is no /tmp on Windows.

Clients should pick up $TMPDIR.

Reply:
Let's please use `Dir.tmpdir` in `bin/rdbg` to make sure the path will be consistent.

We will also need to update the [VS Code extension](https://github.com/Shopify/vscode-ruby-lsp/blob/e1300a6b313c11230c600e944d40f91a747fd29a/src/debugger.ts#L210) to find the same path. To make sure it always matches, I think the most certain way is to run a tiny Ruby script that returns the right path. E.g.:
```typescript
const result = asyncExec("ruby -e 'printf(%{%s/ruby-lsp-debug-sockets}, Dir.tmpdir)'");
const socketsDir = result.stdout;
```

Reply:
Oh, thanks for pointing me to that line https://github.com/Shopify/vscode-ruby-lsp/blob/e1300a6b313c11230c600e944d40f91a747fd29a/src/debugger.ts#L210, i was looking for it, but off course, i have been looking for "/tmp" and not "/", "tmp"

Reply:
In that case, we should be able to use `Dir.tmpdir` in the Ruby side of things and use `$TMPDIR` in the extension. Should make things easy to implement.